src/main/java/com/meilisearch/sdk/SearchRequest.java

```diff
@@ -356,10 +70,10 @@ public String toString() {
         JSONObject jsonObject =
                 new JSONObject()
                         .put("q", this.q)
-                        .put("offset", this.offset)
-                        .put("limit", this.limit)
+                        .put("offset", this.offset == null ? null : this.offset)
```

Comment:
Why do we need this check? If the `this.offset` is `null`, can we just keep `.put("offset", this.offset)` ?

Reply:
The same for the other occurrences!